arch/xtensa/Kconfig

```diff
@@ -81,6 +81,21 @@ config XTENSA_CP_INITSET
 		is provided by CONFIG_XTENSA_CP_INITSET.  Each bit corresponds to one
 		coprocessor with the same bit layout as for the CPENABLE register.
 
+config XTENSA_USE_SEPERATE_IMEM
+	bool "Use a seperate heap for internal memory"
```

Comment:
i guess the description should at least explain:
* what's internal memory
* what uses the memory if this config is enabled

Reply:
I'll add a help section.